src/app/app.config.ts

```diff
@@ -0,0 +1,40 @@
+import { APP_INITIALIZER, ErrorHandler } from '@angular/core';
+import { Router } from '@angular/router';
+import * as Sentry from '@sentry/angular';
+import { environment } from 'src/environments/environment';
+if (environment.production) {
+  Sentry.init({
+    dsn: environment.SENTRY_DNS,
+    integrations: [
+      Sentry.browserTracingIntegration(),
+      Sentry.replayIntegration(),
+    ],
+    tracesSampleRate: 1.0,
```

Comment:
We need to add an attribute of `enabled: process.env.NODE_ENV === 'production'` this would prevent sentry from being activated in localhost